ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -414,7 +410,7 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
     // Update all fractional amounts for groups that already exist
     const updatedGroups = currentHomeOwnerGroups.map(group => {
       const newNumerator = (LCM / group.interestDenominator) * group.interestNumerator
-      group.interestNumerator = newNumerator
+      group.interestNumerator = Math.round(newNumerator)
```

Comment:
Why do ween to round the numerator here?

Reply:
After i removed the code above, this was required to prevent fractions and retain whole numbers after edits. 

The jist, the code above prevented editing down of the `interestDenominator`. It also prevented a new denominator to be defined. 
ie a user wants to delete all groups, define a new one with a lower denominator. Same instance of editing. 

The side of effect of that codes removal was some scenarios where the number would be calculated using a denominator that was not cleanly divisible. 

So this just prevents an error state in the UI (fractional ownership).

Reply:
Ok, thank for clarification. Hopefully, it simplifies fractional ownership, as it is a complex one.

Reply:
It is for sure and to be honest i'm 100% that this is the absolute best solution, since it will change numerator values if a user edits there denominator and so on. 

That being said, it does solve some issues but there seems to be an outlandish amount of edge cases, so won't be surprised if there is more adjustment here 😓